content/javascript/concepts/dates/terms/setUTCMilliseconds/setUTCMilliseconds.md

```diff
@@ -0,0 +1,58 @@
+---
+Title: '.setUTCMilliseconds()'
+Description: 'Sets the milliseconds value of a date object based on Coordinated Universal Time (UTC).'
+Subjects:
+  - 'Web Development'
+  - 'Computer Science'
+Tags:
+  - 'Date'
+  - 'Methods'
+CatalogContent:
+  - 'introduction-to-javascript'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The **`.setUTCMilliseconds()`** method sets the milliseconds value of a `Date` object according to Coordinated Universal Time(UTC) and returns the updated `Date` object.
+
+## Syntax
+
+```pseudo
+myDate.setUTCMilliseconds(millisecondsValue);
+```
+
+The `.setUTCMilliseconds()` method takes a single parameter, representing the numeric value of the milliseconds.
+
+> **Note**: It's important to note that the setUTCMilliseconds() method modifies the Date object in place. It returns the number of milliseconds since January 1, 1970, but it's typically not necessary to use the return value unless it's specifically needed for calculations.
+
+## Example
+
+The JavaScript `Date` object has the `setUTCMilliseconds()` method to set the milliseconds component of a date object in UTC. Here's an example that shows how to use the `setUTCMilliseconds()` method to set the milliseconds of a date:
+
+```javascript
+const date = new Date(Date.UTC(2023, 0, 1, 12, 34, 56, 789));
+
+// Get the current milliseconds in UTC
+const currentUTCMilliseconds = date.getUTCMilliseconds();
+
+// Set a new value for milliseconds (e.g., 2000 milliseconds)
+const newUTCMilliseconds = 2000;
+
+// Create a new Date object with the  date and modified UTC milliseconds
+const newDateWithModifiedMilliseconds = new Date(date);
+newDateWithModifiedMilliseconds.setUTCMilliseconds(newUTCMilliseconds);
+
+console.log('Date:', date.toUTCString());
+console.log('Current UTC Milliseconds:', currentUTCMilliseconds);
+console.log(
+  'Modified Date with UTC Milliseconds:',
+  newDateWithModifiedMilliseconds.toUTCString()
+);
+```
+
+This example results in the following output:
```

Comment:
The format you're using to output the dates doesn't show milliseconds, so the change made in the code isn't visible. Can you alter the example's  output so we see the date with milliseconds.

Reply:
Sure,  I will alter the example.